src/components/common/Directors.vue

```diff
@@ -179,55 +179,56 @@
 
       <!-- Current Director List -->
       <ul class="list director-list">
-        <v-subheader v-if="this.directors.length && !directorEditInProgress" class="director-header">
+        <v-subheader v-if="allDirectors.length && !directorEditInProgress" class="director-header">
           <span>Names</span>
           <span>Delivery Address</span>
           <span v-if="isBComp">Mailing Address</span>
           <span>Appointed/Elected</span>
         </v-subheader>
+        <!-- START FOR LOOP -->
         <li class="director-list-item"
-          v-for="(director, index) in directors"
```

Comment:
Again, too many things named "director".

Since this one is a loop variable, I thought of just using "d" (and "i" for index) but in the end I went with "dir" for the new name. But if we write something like this again, I would go with "d" and "i".